testing/fixtures/fixutres.go

```diff
@@ -0,0 +1,74 @@
+package fixtures
+
+import (
+	"os"
+	"path/filepath"
+	"sync"
+
+	"github.com/fossas/fossa-cli/files"
+	git "gopkg.in/src-d/go-git.v4"
+	"gopkg.in/src-d/go-git.v4/plumbing"
+)
+
+// ProjectFixture describes a git locator for a test fixture and env vars for when it is to be built
+type ProjectFixture struct {
```

Comment:
I'd just call this `Project` since it's already in `package fixtures` (users will consume it as `fixtures.Project` which sounds better than `fixtures.ProjectFixture`).